### PR TITLE
Prepare for 3.2.5

### PR DESCRIPTION
**Pull request by Ayyanchira**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,6 +15,16 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 #### Fixed
 - nothing yet
 
+
+## [3.2.5](https://github.com/Iterable/iterable-android-sdk/releases/tag/3.2.5)
+#### Added
+- Introducing authToken. `setUserId` and `setEmail` methods now takes optional `authToken` parameter
```
Comment (Ayyanchira) [inline CHANGELOG.md:21]: @vbabenkoru 
Do you think we should include this only when backend is ready? and everything is tested?

Comment (vbabenkoru) [reply CHANGELOG.md:21]: We should make the methods private and ship 3.2.5 without this note. Once everything is ready for JWT, make them public again, ship 3.2.6 and add a link to the JWT docs in the changelog.

File: CHANGELOG.md [hunk 2]
```diff
@@ -15,6 +15,16 @@ This project adheres to [Semantic Versioning](http://semver.org/).
...
+- `app_name` is set to `IterableAPI` in strings resource file
```
Comment (vbabenkoru) [inline CHANGELOG.md:22]:
```suggestion
- `app_name` was removed from published strings and replaced with plain string values in test manifests.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -15,6 +15,16 @@ This project adheres to [Semantic Versioning](http://semver.org/).
...
+
+#### Fixed
+- Fixed an issue where trackInAppClicks were not being registered after displaying an inApp.
```
Comment (vbabenkoru) [inline CHANGELOG.md:25]:
```suggestion
- Fixed an issue where in-app click events were not being registered after displaying an in-app message.
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -15,6 +15,16 @@ This project adheres to [Semantic Versioning](http://semver.org/).
...
+- Fixed crash issues by handling null cases caused in `IterableInAppFragmentHTMLNotification` class
```
Comment (vbabenkoru) [inline CHANGELOG.md:26]:
```suggestion
- Fixed NullPointerExceptions in `IterableInAppFragmentHTMLNotification` that could occur in some cases when the activity is destroyed and recreated.
```

File: CHANGELOG.md (hunk 1 above)
Comment (Ayyanchira) [reply CHANGELOG.md:21]: Should I make those methods private in this PR? OR should I have it seperate and rebase this one?

Comment (vbabenkoru) [reply CHANGELOG.md:21]: I think ideally a separate PR, but here is also ok.

File: CHANGELOG.md [hunk 5]
```diff
@@ -15,6 +15,15 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 #### Fixed
 - nothing yet
 
+
+## [3.2.5](https://github.com/Iterable/iterable-android-sdk/releases/tag/3.2.5)
+#### Added
+- `app_name` was removed from published strings and replaced with plain string values in test manifests.
```
Comment (vbabenkoru) [inline CHANGELOG.md:21]: I think this should be in 'Changed' rather than 'Added'.

Comment (Ayyanchira) [reply CHANGELOG.md:21]: oh yes

File: CHANGELOG.md [hunk 6]
```diff
@@ -15,6 +15,15 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 #### Fixed
 - nothing yet
 
+
+## [3.2.5](https://github.com/Iterable/iterable-android-sdk/releases/tag/3.2.5)
+#### Added
```
Comment (vbabenkoru) [inline CHANGELOG.md:20]:
```suggestion
#### Changed
```

**Pull request merged**